app/src/test/java/com/eeyan/cryptapp/data/mappers/DtoToDomainKtTest.kt

```diff
@@ -0,0 +1,78 @@
+package com.eeyan.cryptapp.data.mappers
+
+import com.eeyan.cryptapp.data.remote.dto.CoinDetailDto
+import com.eeyan.cryptapp.data.remote.dto.CoinDto
+import com.eeyan.cryptapp.data.remote.dto.Whitepaper
+import com.eeyan.cryptapp.data.utils.LinksExtendedUtil
+import com.eeyan.cryptapp.data.utils.TeamMemberUtil
+import com.eeyan.cryptapp.data.utils.links
+import com.eeyan.cryptapp.domain.model.Coin
+import com.eeyan.cryptapp.domain.model.CoinDetail
+import com.google.common.truth.Truth.assertThat
+import org.junit.Test
+
+
+
+class DtoToDomainKtTest {
+
+    @Test
+    fun `confirm that CoinDto toCoin results to Coin`() {
+        val coinDto = CoinDto(
+            id = "android",
+            is_active = false,
+            is_new = true,
+            name = "name",
+            rank = 10,
+            symbol = "symbol",
+            type = "type"
+        )
```

Comment:
To shorten the number of lines in the test class, you could also shift the constants to TestUtils

Reply:
Working on it ✊